resources/files/rbconfig-s390x-linux-gnu.rb

```diff
@@ -0,0 +1,268 @@
+
+# This file was created by PuppetAgent for HuaweiOS (Debian 8 PPC).  Any
+# changes made to this file will be lost the next time ruby is built.
```

Comment:
This file is generated by building Ruby on the platform (minus the cross compile, have we done that?)